bokeh/document.py

```diff
@@ -336,6 +350,10 @@ def to_json_string(self):
             }
         }
 
+        custom_models = self._collect_custom_models()
+        if custom_models:
+            json['models'] = custom_models
+
```

Comment:
another advantage of doing it in `_standalone_docs_json_and_render_items` is that if there's multiple docs we only send the models once